hack/Dockerfile

```diff
@@ -283,7 +296,75 @@ ENTRYPOINT ["pyenv", "exec", "python3", "-m", "gunicorn", "--config=/src/src/das
 
 # -----------------------------------------------------------------------------
 
-FROM base AS archivematica-tests
+FROM base-builder as archivematica-tests
+
+ARG USER_ID=1000
+ARG GROUP_ID=1000
+ARG PYENV_DIR=/pyenv
+
+RUN set -ex \
+	&& curl --retry 3 -fsSL https://packages.archivematica.org/1.16.x/key.asc | gpg --dearmor -o /etc/apt/keyrings/archivematica-1.16.x.gpg \
+	&& echo "deb [arch=amd64 signed-by=/etc/apt/keyrings/archivematica-1.16.x.gpg] http://packages.archivematica.org/1.16.x/ubuntu-externals jammy main" > /etc/apt/sources.list.d/archivematica-external.list \
+	&& curl --retry 3 -so /tmp/repo-mediaarea_1.0-21_all.deb -L https://mediaarea.net/repo/deb/repo-mediaarea_1.0-21_all.deb \
+	&& dpkg -i /tmp/repo-mediaarea_1.0-21_all.deb \
+	&& apt-get update \
+	&& apt-get install -y --no-install-recommends \
+		atool \
+		bulk-extractor \
+		clamav \
+		coreutils \
+		ffmpeg \
+		fits \
+		g++ \
+		gcc \
+		gearman \
+		gettext \
+		ghostscript \
+		hashdeep \
+		imagemagick \
+		inkscape \
+		jhove \
+		libffi-dev \
+		libimage-exiftool-perl \
+		libldap2-dev \
+		libmysqlclient-dev \
+		libsasl2-dev \
+		libssl-dev \
+		libxml2-dev \
+		libxslt1-dev \
+		logapp \
+		md5deep \
+		mediaconch \
+		mediainfo \
+		nailgun \
+		nfs-common \
+		openjdk-8-jre-headless \
+		p7zip-full \
+		pbzip2 \
+		pst-utils \
+		rsync \
+		siegfried \
+		sleuthkit \
+		tesseract-ocr \
+		tree \
+		unar \
+		unrar-free \
+		uuid \
+	&& rm -rf /var/lib/apt/lists/* /var/cache/apt/*
+
+RUN set -ex \
+	&& groupadd --gid ${GROUP_ID} --system archivematica \
+	&& useradd --uid ${USER_ID} --gid ${GROUP_ID} --home-dir /var/archivematica --system archivematica \
+	&& mkdir -p /var/archivematica/sharedDirectory \
+	&& chown -R archivematica:archivematica /var/archivematica
+
+# Download ClamAV virus signatures
+RUN freshclam --quiet
+
+USER archivematica
+
+COPY --chown=${USER_ID}:${GROUP_ID} --from=pyenv-builder --link ${PYENV_DIR} ${PYENV_DIR}
+COPY --chown=${USER_ID}:${GROUP_ID} --link . /src
```

Comment:
Not sure if it makes sense, but I think if you end up moving all the OS dependencies out of `base` to their corresponding stages, this block might be inherited from the `base` stage instead.

Reply:
So what I think you are saying is that this block is not really needed because it already happens in `base`. So by having this block here is just redundant, doing the same work again, right?

Reply:
That's correct.

Reply:
Done.